docs/src/guide/discretisation.jl

```diff
@@ -198,22 +199,121 @@ Nrange = 10:10:100
 plot(Nrange, abs.(fconv.(Nrange) .- fconv(200)); yaxis=:log, legend=false)
 
 # ### Exercise 2
-# !!! note "TODO"
-#     This solution has not yet been written. Any help with a PR is appreciated.
+#      ```math
+#         \bullet \langle e_G, e_{G'}\rangle = ∫_0^{2π} e_G^\ast(x) e_{G'}(x) d x = 1/2π ∫_0^{2π} e^i(G'-G)x d x
```

Comment:
Why `\bullet` ?

Reply:
I thought I would put a bullet for each part of the equation, should I remove it ?